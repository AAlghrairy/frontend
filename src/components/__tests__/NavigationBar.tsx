import * as React from 'react'

import { shallow } from 'enzyme'

import NavigationBar from '../NavigationBar'

test('NavigationBar renders correctly', () => {
  const tree = shallow(<NavigationBar title="Cadet" />)
  expect(tree.debug()).toMatchSnapshot()
})
