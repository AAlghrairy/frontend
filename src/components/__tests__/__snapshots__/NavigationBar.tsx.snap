// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar renders correctly 1`] = `
"<Blueprint2.Navbar className=\\"NavigationBar pt-dark\\">
  <Blueprint2.NavbarGroup align=\\"left\\">
    <Blueprint2.NavbarHeading>
      Cadet
    </Blueprint2.NavbarHeading>
    <Blueprint2.NavbarDivider />
    <NavLink to=\\"/dashboard\\" activeClassName=\\"pt-active\\" className=\\"NavigationBar__link pt-button pt-minimal\\" ariaCurrent=\\"true\\">
      <Blueprint2.Icon icon=\\"dashboard\\" />
      Dashboard
    </NavLink>
  </Blueprint2.NavbarGroup>
</Blueprint2.Navbar>"
`;
